packages/meta/galaxy/project_galaxy_util.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+__version__ = '19.9.0.dev0'
+
+PROJECT_NAME = "galaxy"
+PROJECT_OWNER = PROJECT_USERAME = "galaxyproject"
+PROJECT_URL = "https://github.com/galaxyproject/galaxy"
+PROJECT_AUTHOR = 'Galaxy Project and Community'
+PROJECT_DESCRIPTION = 'Galaxy Server Metapackage'
+PROJECT_EMAIL = 'jmchilton@gmail.com'
```

Comment:
We switched this to ``galaxy-committers@lists.galaxyproject.org``

Reply:
And here I missed the chance to harass you about claiming authorship of all of Galaxy's code